db/migrate/20150405012126_remove_approval_status_from_reservation.rb

```diff
@@ -0,0 +1,44 @@
+class RemoveApprovalStatusFromReservation < ActiveRecord::Migration
```

Comment:
Did you test this at all? Why don't you check out `master`, quickly create all the different types of reservations (you can use `FactoryGirl` in the console), and make sure that everything is flagged and status-ed as expected.